src/app.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 const session = require('express-session');
 const passport = require('passport');
 require('dotenv').config();
+const swaggerUi = require('swagger-ui-express');
+const swaggerDocument = require('./API-documentation.json');
```

Comment:
It's better to avoid uppercase letters in file naming, so you can rename this to `api-documentation.json`

Reply:
Also move this line below `const authRouter = require('./routes/auth');` so that package require statements and project code require statements are separated.

Reply:
@Shrreya after moving the swagger requirement lines below `const authRouter = require('./routes/auth');` line i got an error from automated test says that i should move `swagger-ui-express` line before `const Database = require('./db');` line, so i revert the change again and the automated tests passed correctly

Reply:
Okay no problem, it's not a big issue.

Reply:
@Shrreya done!